layouts/partials/head.html

```diff
@@ -0,0 +1,3 @@
+{{- if .Page.Store.Get "hasDiagramsNet" -}}
```

Comment:
We should load this diagram loader in a different partial since creating a partial named head overwrites the original from docsy which is being referenced here:
https://github.com/sapcc/hugo-documentation-templater/blob/main/layouts/index.html#L5